gitpod-io/openvscode-server
# Enable all proposed apis

Enable all proposed APIs for all extensions

## How to Test
- Open workspace in prev env https://hw-proposal.preview.gitpod-dev.com/workspaces with latest code selected
- See if `OUTPUT` section in terminal area have no proposal APIs missing like below

```
CRITI Extension 'ms-vscode.js-debug' appears in product.json but enables LESS API proposals than the extension wants.
package.json (LOSES): portsAttributes, findTextInFiles, workspaceTrust, resolvers, tunnels
product.json (WINS): portsAttributes, findTextInFiles, workspaceTrust, resolvers
```
